src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -613,6 +627,72 @@ public Response getRsync(@PathParam("identifier") String id) {
             return error(Response.Status.INTERNAL_SERVER_ERROR, "Something went wrong attempting to download rsync script: " + EjbUtil.ejbExceptionToString(ex));
         }
     }
+    
+    @POST
+    @Path("{identifier}/dataCaptureModule/checksumValidation")
+    public Response receiveChecksumValidationResults(@PathParam("identifier") String id, JsonObject jsonFromDcm) throws IOException {
```

Comment:
Whoops. I'll take that out as well as some imports I added when adding test code. Good catch, @oscardssmith . Thanks!